src/components/uploader/css/Uploader.css

```diff
@@ -485,6 +485,10 @@ html>body .fl-uploader span.fl-uploader-browse {
     display:block;
 }
 
+.fl-uploader-no-js-note {
```

Comment:
The expanse of your wiki is quite daunting to look at first :sweat_smile: I'll change this in a while.